ckcp/openshift_dev_setup.sh

```diff
@@ -275,20 +266,18 @@ patches:
         value: $ckcp_route " >>"$ckcp_temp_dir/kustomization.yaml"
 
   echo -n "  - kcp $kcp_version: "
-  kubectl apply -k "$ckcp_temp_dir" >/dev/null
+  # Deploy ckcp until all resources are successfully appied to OCP cluster 
+  local i=0
+  while ! error_msg=$(kubectl apply -k "$ckcp_temp_dir" 2>&1 1>/dev/null); do
+    sleep 10
+    i=$((i+1))
+    if [ $i -gt 12 ]; then
+      printf "\n Failed to deploy ckcp \n"
```

Comment:
Here there is a loop and you need to do the apply one more time without redirecting to /dev/null
(See my  comment below). Only comment and exit if the latest apply fails

Reply:
sorry I missed that you were storing the output in error_msg. Ignore my previous comment.

Reply:
discussed with @fgiloux , we can keep it this way